README.md

```diff
@@ -1,2 +1,24 @@
-# Functional.CQS
-Defines CQS-style contracts and provides AOP facilities over those contracts using [Functional](https://github.com/JohannesMoersch/Functional) primitives
+# Functional.CQS.AOP
+
+`Functional.CQS.AOP` is a set of NuGet packages that provide decoration facilities over implementations of [`Functional.CQS` handler contracts](https://github.com/RyanMarcotte/Functional.CQS/tree/master/src/Functional.CQS), specifically for [aspect-oriented programming](https://en.wikipedia.org/wiki/Aspect-oriented_programming) / applying [cross-cutting concerns](https://stackoverflow.com/questions/23700540/cross-cutting-concern-example).
+
+The following contracts are defined:
```

Comment:
Since these contracts all come from `Functional.CQS`, this list can be moved to the `Functional.CQS` section, before you move on to `Functional.CQS.AOP`.

Reply:
Preemptively done! :)